src/GitVersion.Core/Git/SemanticVersionBuildMetaData.cs

```diff
@@ -87,15 +87,15 @@ public string ToString(string? format, IFormatProvider? formatProvider)
         if (formatProvider?.GetFormat(GetType()) is ICustomFormatter formatter)
             return formatter.Format(format, this, formatProvider);
 
-        if (format.IsNullOrEmpty())
+        if (string.IsNullOrEmpty(format))
```

Comment:
I think `format.IsNullOrEmpty()` reads better than `string.IsNullOrEmpty(format)`. Why is it changed?

Reply:
I have changed it back